pkg/api/types.go

```diff
@@ -886,6 +1324,41 @@ type PodStatus struct {
 	// Shows the status of the init Units on the Pod with one entry for each
 	// init Unit in the Pod's Spec.
 	InitUnitStatuses []UnitStatus `json:"initUnitStatuses"`
+
+	// NOT SUPPORTED YET, ADDED FOR CONFORMANCE
+	// +optional
+	Conditions []core.PodCondition `json:"conditions,omitempty"`
+	// A human readable message indicating details about why the pod is in this state.
+	// +optional
+	Message string `json:"message,omitempty"`
+	// A brief CamelCase message indicating details about why the pod is in this state. e.g. 'Evicted'
+	// +optional
+	Reason string `json:"reason,omitempty"`
+	// nominatedNodeName is set when this pod preempts other pods on the node, but it cannot be
+	// scheduled right away as preemption victims receive their graceful termination periods.
+	// This field does not guarantee that the pod will be scheduled on this node. Scheduler may decide
+	// to place the pod elsewhere if other nodes become available sooner. Scheduler may also decide to
+	// give the resources on this node to a higher priority pod that is created after preemption.
+	// +optional
+	NominatedNodeName string
+	// +optional
+	HostIP string
+
+	// PodIPs holds all of the known IP addresses allocated to the pod. Pods may be assigned AT MOST
+	// one value for each of IPv4 and IPv6.
+	// +optional
+	PodIPs []core.PodIP `json:"podIPs,omitempty"`
+
+	// Date and time at which the object was acknowledged by the Kubelet.
+	// This is before the Kubelet pulled the container image(s) for the pod.
+	// +optional
+	StartTime *metav1.Time `json:"startTime,omitempty"`
+	// +optional
+	QOSClass core.PodQOSClass `json:"QOSClass,omitempty"`
```

Comment:
This should be relatively easy to implement, so we can copy over the type definition.